index.bs

```diff
@@ -2140,7 +2153,25 @@ Base64-encoded string.
       }
 
       browsingContext.CaptureScreenshotParameters = {
-        context: browsingContext.BrowsingContext
+        context: browsingContext.BrowsingContext,
+        ?clip: browsingContext.ClipRectangle
+      }
+
+      browsingContext.ClipRectangle = (browsingContext.ElementClipRectangle //
+                                       browsingContext.BoxClipRectangle)
+
+      browsingContext.ElementClipRectangle = {
+        type: "element",
+        element: script.SharedReference,
+        ?scrollIntoView: bool
```

Comment:
```suggestion
        ? scrollIntoView: bool
```

Question: Do we actually capture the full element when parts of it are outside the visible viewport? Or would be the `full` screenshot support necessary for that?

Reply:
No, we always clip to the viewport. I don't know if that's the behaviour of classic, but if you want the full visible part of the element in the future we'll add full-document screenshot support and combining _that_ with an element clip will get you the full element (or at least any part of it that's visible).

Reply:
Ok, that sounds fine with me.